test/player_test.rb

```diff
@@ -7,19 +7,17 @@ def setup
     @player = Noplayer.new
   end
 
-
   # Default name
 
   def test_default_name
-    assert_equal "Noplayer", @player.name
+    assert_equal 'Noplayer', @player.name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.